cvat/apps/engine/cache.py

```diff
@@ -312,7 +312,7 @@ def _prepare_cloud_preview(self, db_storage):
             manifest.set_index()
```

Comment:
Not really because without this optimization it was impossible to create a task from the cloud storage with `use_cahce==True` with a huge dataset because the background job would crash due to reaching the memory limit.